chxtools/chx_wrapper.py

```diff
@@ -43,30 +44,32 @@ def cw_ascan(mot,xmin,xmax,npoints,acqt='default',pos_ret=True):
     WILL NOT WORK FOR A LIST OF DETECTORS!
     """
     # gather beamline information prior to starting the scan:
-    ini_motpos=caget(mot.record+'.RBV')
+    ini_motpos = caget(mot.record+'.RBV')
```

Comment:
`caget` shouldn't be used in a plan. @mrakitin Can you open an issue for this too?